src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -65,7 +65,7 @@ const MainLayout = async (props: Props) => {
             </button>
             <UserMenu
               session={session}
-              fxaSettingsUrl={AppConstants.NEXT_PUBLIC_FXA_SETTINGS_URL}
+              fxaSettingsUrl={AppConstants.FXA_SETTINGS_URL}
```

Comment:
We are currently using `NEXT_PUBLIC_FXA_SETTINGS_URL` in the [UserMenu](https://github.com/mozilla/blurts-server/blob/main/src/app/components/client/toolbar/UserMenu.tsx) component, which is a client component.

Reply:
Right but doesn't it get evaluated in the server component?